paper-card.html

```diff
@@ -237,6 +237,10 @@
         return image ? 'false' : 'true';
       },
 
+      created: function() {
+        this.classList.add('paper-material');
```

Comment:
There's still a `ready` - the only documented change is that light DOM distribution is no longer guaranteed to be done before `ready` (https://github.com/Polymer/polymer/tree/2.0-preview#distribution)